### DISCUSSION TITLE
Windows 10 IoT (for ARM) on Harman Kardon Invoke

### DISCUSSION THREAD

**Discussion by WinInsider** [coggy9/HKHacking#12] [Q&A] [answered]
Could the compute daughterboard (LS9ADAC11DBT, based on Coretex-A7) inside Harman Kardon Invoke **boot** to [Windows 10 IoT ARM](https://github.com/ms-iot/iot-adk-addonkit)?

I would imagen that finding drivers (Bluetooth, WiFi and audio) would be a challenge...

**Comment by coggy9** [comment, marked as answer]
With only 256MB of flash storage, AFAIK there is not enough storage to try it unless you find a bigger compatible NAND chip and replace the default one. Even then, I doubt it would work.
